ludwig/encoders/sequence_encoders.py

```diff
@@ -208,7 +213,11 @@ def __init__(
             embedding_regularizer=weights_regularizer
         )
 
-    def call(self, inputs, training=None, mask=None):
+    def get_output_shape(self, input_shape):
+        # assume input shape (seq_size, )]\
+        return input_shape[0], self.embedding_size
```

Comment:
shouldn't this be a torch.Size ?

Reply:
corrected when overriding the both the input_shape and output_shape address issue of "fake" tensor not being the correct dtype for sequence encoders.